runtests.py

```diff
@@ -183,7 +183,7 @@ def main(argv):
     if args.shell:
         shell = os.environ.get('SHELL', 'sh')
         print("Spawning a Unix shell...")
-        os.execv(shell, [shell] + extra_argv)
+        subprocess.check_call([shell] + extra_argv)
         sys.exit(1)
```

Comment:
Yep, before it was there in case if fails. Now it's required, but I guess a `return` might be more clear.